bigquery/simpleapp/simpleapp.go

```diff
@@ -55,6 +55,7 @@ func query(proj string) (*bigquery.RowIterator, error) {
 	if err != nil {
 		return nil, err
 	}
+	defer client.Close()
```

Comment:
This returns a `*bigquery.RowIterator`. Are we sure we can close the client when this returns? Also, the comment says it returns a slice, but it doesn't.

Reply:
Refactored simpleapp so we construct client in main and it persists through the lifecycle.

Reply:
And addressed comment.